Marlin/Configuration_adv.h

```diff
@@ -1504,68 +1496,122 @@
 
 #endif // HAS_TRINAMIC
 
-// @section L6470
+// @section ST_L6470
 
 /**
- * L6470 Stepper Driver options
+ * ST_L6470 Stepper Driver options
+ *
+ * The Arduino-6470 library (revision 0.7.0 and higher)  is required for this stepper driver.
+ * https://github.com/ameyer/Arduino-6470
```

Comment:
The library is "`Arduino-L6470`" — without the "L" the name is incorrect and the link won't work.